### ISSUE TITLE
Atom.Object.defineProperty.get is deprecated.

### ISSUE THREAD

**Issue by StatelessCat** [uzyn/tag#5]
atom.workspaceView is no longer available.
In most cases you will not need the view. See the Workspace docs for
alternatives: https://atom.io/docs/api/latest/Workspace.
If you do need the view, please use `atom.views.getView(atom.workspace)`,
which returns an HTMLElement.
```
Atom.Object.defineProperty.get (/usr/share/atom/resources/app/src/atom.js:54:11)
Object.activate (/home/raphael/.atom/packages/tag/lib/tag.js:11:9)
```

**Comment by StatelessCat** [comment]
Not sure how to convert this, but this post can help : [Replacement for atom.workspaceView.command](https://discuss.atom.io/t/replacement-for-atom-workspaceview-command/13636)

**Comment by sbromberger** [comment]
duplicate of #4

**Comment by uzyn** [comment]
This issue should be fixed now with v0.3.0.

Please update.
